tools/build-webview

```diff
@@ -62,6 +58,14 @@ elif [ -z "${DEST-}" ]; then
     usage; exit 2
 fi
 
+# If DEST is relative (or contains symlinks), make it absolute for the
+# sake of the glob checks below.
+DEST=$(readlink -m "$DEST")
```

Comment:
`readlink` doesn't appear to take a `-m` option on OS X.

Reply:
Oof, thanks. :-/ I'll go looking for a workaround, though not today.